### ISSUE TITLE
[Feature Request] Support for loading client-side type of Fabric mods to the Bedrock users who join the servers via GeyserMC

### ISSUE THREAD

**Issue by joeyoropesa-dev** [BukkitFabric/bukkit4fabric#400]
It would be nice to make this to be possible because that means that most of mods that already exist can without re-writting from scratch and etc... we can enjoy with full server experience with mods even if we joined with vanilla Minecraft Bedrock Edition via GeyserMC

**Comment by MagmaSlime123** [comment]
Modding on Bedrock is pretty much impossible and (with how dumb/stupid/strict Microsoft is) not allowed.

**Comment by admin-elvistrujanovic** [comment]
> Modding on Bedrock is pretty much impossible and (with how dumb/stupid/strict Microsoft is) not allowed.

So you want to tell me that this project is officially dead due to this reason?

**Comment by joeyoropesa-dev** [comment]
> Modding on Bedrock is pretty much impossible and (with how dumb/stupid/strict Microsoft is) not allowed.

Yeah, you're right but what if custom developers can bypass those restrictions and APIs finding some alternatives for the functions, features and etc... that Java used for client-side mods to work?

**Comment by MagmaSlime123** [comment]
> > Modding on Bedrock is pretty much impossible and (with how dumb/stupid/strict Microsoft is) not allowed.
> 
> Yeah, you're right but what if custom developers can bypass those restrictions and APIs finding some alternatives for the functions, features and etc... that Java used for client-side mods to work?

Again, you should know how trigger-happy Mojang and Microsoft both are with Bedrock modding. Don't do it. Period.

**Comment by User8395** [comment]
PolyMC exists which allows clients to join without requiring mods installed.

**Comment by GeraldTM** [comment]
> PolyMC exists which allows clients to join without requiring mods installed.

Lol wtf you talking about, PolyMC is a dead minecraft launcher, and all it did was allow you to create multiple instances of the game and install mods through the launcher

**Comment by User8395** [comment]
Not that PolyMC

[This PolyMC](https://github.com/TheEpicBlock/PolyMc)
